xamarin/Xamarin.Forms
# [GH-7754] - TitleView's Background Color

### Description of Change ###

Added an Attached Bindable Property to NavigationPage, to control the BarBackgroundColor.

### Issues Resolved ### 

- fixes #7754 

### API Changes ###


Added:
- `public static readonly BindableProperty NavigationPage.BackgroundTitleViewProperty = BindableProperty.CreateAttached`
- `public event EventHandler UpdateBackgroundTitleView;`

- `public void FireUpdateBackgroundTitleViewEvent() =>
			UpdateBackgroundTitleView?.Invoke(null, null);`


### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###
Now the user can set the BarBackground from a Page.

### Before/After Screenshots ### 


### Testing Procedure ###
Go to the Issue7754 page, and press the button. The BarBackground color should change when you click on the button.

### PR Checklist ###
